bip-taro.mediawiki

```diff
@@ -532,17 +530,20 @@ where:
 * <code>prev_asset_witnesses</code>: is a ''nested'' TLV that contains the asset witnesses needed to verify the merging into the target asset leaf
 ** <code>prev_asset_id</code>: references the previous asset input by the input position within the transaction, the asset ID of the previous asset tree, and the asset script hash.
 *** This value is included in any signatures generated within the asset witness itself, serving a purpose similar to the previous outpoint in vanilla Bitcoin.
-*** If this field is all zeroes, then that indicates a ''new'' asset is being created.
+*** If the <code>asset_script_key</code> field is all zeroes, then that indicates a ''new'' asset is being created.
```

Comment:
Need to also tie this to an asset TLV version change if we decide it's actually necessary. 

Also how can the script key be all zeroes when we need to sign with that (or reveal a tapscript path) to be able to move it again?

Reply:
This refers to the _prev_ `asset_script_key`, from `prev_asset_id = prev_outpoint || asset_id || asset_script_key`.

Renamed it to `prev_asset_script_key` for clarity: https://github.com/Roasbeef/bips/pull/33/commits/57ee255ac988667f84a531687f039d3d6fa86d86